.zuul.yaml

```diff
@@ -0,0 +1,75 @@
+- job:
+    name: rust-base
+    parent: init-test
+    description: |
+      Base job for all types of rust test jobs.
+    pre-run: playbooks/rust-base/pre.yaml
+    vars:
+      rustup_toolchain: stable
+    nodeset: ubuntu-bionic
+
+- job:
+    name: rust-osauth-exercises
+    parent: rust-base
+    description: |
+      Run rust-openstack acceptance test on master branch
+    run: playbooks/rust-osauth-exercises.yaml
+
+- job:
+    name: rust-osauth-exercises-train
+    parent: rust-osauth-exercises
+    description: |
+      Run rust-openstack acceptance test on train branch
+    vars:
+      global_env:
+        OS_BRANCH: stable/train
+
+- job:
+    name: rust-osauth-exercises-stein
+    parent: rust-osauth-exercises
+    description: |
+      Run rust-openstack acceptance test on stein branch
+    vars:
+      global_env:
+        OS_BRANCH: stable/stein
+    nodeset: ubuntu-xenial
+
+- job:
+    name: rust-osauth-exercises-rocky
+    parent: rust-osauth-exercises
+    description: |
+      Run rust-openstack acceptance test on rocky branch
+    vars:
+      global_env:
+        OS_BRANCH: stable/rocky
+    nodeset: ubuntu-xenial
+
+- job:
+    name: rust-osauth-exercises-queens
+    parent: rust-osauth-exercises
+    description: |
+      Run rust-openstack acceptance test on queens branch
+    vars:
+      global_env:
+        OS_BRANCH: stable/queens
+    nodeset: ubuntu-xenial
+    voting: false
+
+- job:
+    name: rust-osauth-exercises-nightly
+    parent: rust-base
+    description: |
+      Run rust-openstack acceptance test on master branch with nightly rust.
+    run: playbooks/rust-osauth-exercises.yaml
+    vars:
+      rustup_toolchain: nightly
+    voting: false
+- project:
+    check:
```

Comment:
name is missing.

```
- project:
    name: dtantsur/rust-osauth
```

Reply:
Is it a requirement of openlab? In openstack we have removed usages of `name` to make jobs reusable.

Reply:
emm, I'm not sure, since we didn't upgrade  the zuul in openlab for long time.

The job define looks good. Can you merge this PR first? I can debug it next to make sure the job works.